### PR TITLE
Use ::class instead of string class names

### PR DESCRIPTION
**Pull request by mvorisek**
Fixes #174 
Fixes #180

### CODE REVIEW

File: tests/OracleTest.php [hunk 1]
```diff
@@ -33,8 +33,8 @@ public function connect($ver = '')
     {
         return new \atk4\dsql\Connection(array_merge([
             'connection'       => new \PDO('sqlite::memory:'),
-            'query_class'      => 'atk4\dsql\Query_Oracle'.$ver,
-            'expression_class' => 'atk4\dsql\Expression_Oracle',
```
Comment (mvorisek) [inline tests/OracleTest.php:37]: `\atk4\dsql\Expression_Oracle` class seems to be even not presented

Comment (DarkSide666) [reply tests/OracleTest.php:37]: Should remove that expression_class line (will use default Expression class then), but query_class line should be here!

Comment (mvorisek) [reply tests/OracleTest.php:37]: done, of course - the first one was refactored only, but GH is showing 1 red/removed line and 1 added

File: tests/ExceptionTest.php [hunk 2]
```diff
@@ -57,9 +51,17 @@ public function testNonexistantFieldException()
...
+        $q->table('dual')->field('do_not_exist');
+
+        $this->setExpectedException(\atk4\dsql\ExecuteException::class);
 
-        $this->setExpectedException('atk4\dsql\ExecuteException');
-        $q->execute(); // PDOException: SQLSTATE[HY000]: General error: 1 no such table: foo
+        try {
+            $q->execute();
+        } catch (\atk4\dsql\ExecuteException $e) {
+            $this->assertSame(1054, $e->getCode());
```
Comment (mvorisek) [inline tests/ExceptionTest.php]: This is my idea to test, but still do not know how to query MySQL/MariaDB here

Comment (DarkSide666) [reply tests/ExceptionTest.php]: How about using `driver` property? It's `sqlite`, `mysql` or `pgsql` or `oracle` - taken from Persistence class.

**Review by DarkSide666 — APPROVED** [review]
LGTM

File: tests/ExceptionTest.php (hunk 2 above)
Comment (mvorisek) [reply tests/ExceptionTest.php]: Why not, but I can not really open hundreds of PRs for these smaller fixes.

**Pull request merged**